.github/workflows/Updater.yml

```diff
@@ -27,8 +27,16 @@ jobs:
           ruby-version: '3.3.6'
       - name: Install Python pip
         run: sudo apt install -y python3-pip
-      - name: Install activesupport
-        run: sudo apt install -y ruby-activesupport
+      - name: Install ruby-libversion  # Hopefully this will get added as an Ubuntu/Debian package so we don't have to do this manually.
+        run: |
+          git clone https://github.com/repology/libversion
+          cd libversion
+          mkdir build
+          cd build
+          cmake ..
+          make -j $(nproc)
+          sudo make install
+          sudo gem install ruby-libversion
```

Comment:
How long does this build process take?

Reply:
probably about 30 seconds, but i'll swap it to a shallow clone which should cut the total time to around 15

Reply:
Do you know how to create an Ubuntu ppa for a debian package? That would the easiest way of getting a pre-built Ubuntu package here... Well, maybe that would only save ~ 15 seconds...

Reply:
neither ubuntu nor debian package ruby-libversion or the dependent libversion library, so we'd be starting from scratch. I looked into adding those packages to debian, but its more trouble than its worth, imo.